### DISCUSSION TITLE
"lower_bound" or "<diag_name>.diag_lo" don't work correctly.

### DISCUSSION THREAD

**Discussion by jiecaix** [BLAST-WarpX/warpx#4439] [Q&A] [answered]
# Question description
I use a lower_bound in the Field diagnostic, the global range of x direction is "-250um ~ 0um". And I set the lower bound in x direction of "-100 um", but the range I get is "-250 ~ -150um", and when I change the upper bound to "-5um", the real boundary change to be "-250 ~ -245 um", but the upper bound cut the real bound should to be "-5um ~ 0um".

## input file:

##algo
algo.maxwell_solver = Yee
algo.particle_shape = 3

##amr
amr.max_grid_size = 32
amr.max_level = 0
amr.n_cell = 3600 800

##boundary
boundary.field_hi = pml pml
boundary.field_lo = pml pml
boundary.particle_hi = absorbing absorbing
boundary.particle_lo = absorbing absorbing

##diagnostics
diagnostics.diags_names = species field wall_field

##geometry
geometry.dims = 2
geometry.prob_hi = 0.0 0.0001
geometry.prob_lo = -0.00025 -0.0001

##wall_field
wall_field.diag_lo = -0.0001 -0.0001 -0.0001
wall_field.diag_type = Full
wall_field.fields_to_plot = Ex Ey Ez
wall_field.file_prefix = ./WallFieldDiagnostic/
wall_field.format = openpmd
wall_field.intervals = 5
wall_field.openpmd_backend = h5
wall_field.write_species = 0

##warpx
warpx.cfl = 0.95
warpx.do_dive_cleaning = 0
warpx.do_moving_window = 1
warpx.moving_window_dir = x
warpx.moving_window_v = 1.0
warpx.serialize_initial_conditions = 1
warpx.start_moving_window_step = 0
warpx.use_filter = 1
warpx.verbose = 1

# Result of wall_field
![image](https://github.com/ECP-WarpX/WarpX/assets/56915687/b239532c-d995-45ac-8ffb-d73c7331fc84)

**Comment by ax3l** [comment, marked as answer]
Moved to an issue: https://github.com/ECP-WarpX/WarpX/issues/4435

Closing here.
